frontend/server/src/Controllers/Run.php

```diff
@@ -914,10 +941,28 @@ public static function apiSource(\OmegaUp\Request $r): array {
         return self::getOptionalRunDetails(
             $submission,
             $run,
-            false
+            /*$showDetails=*/ false
         );
     }
 
+    private static function shouldShowRunDetails(
+        int $identityId,
+        \OmegaUp\DAO\VO\Problems $problem,
+        ?\OmegaUp\DAO\VO\Contests $contest
+    ): string {
+        $isProblemSolved = \OmegaUp\DAO\Problems::isProblemSolved(
+            $problem,
+            $identityId
+        );
+        if (
+            !is_null($contest)
+            && $contest->finish_time->time > \OmegaUp\Time::get()
```

Comment:
nit:

```suggestion
            !is_null($contest) &&
            $contest->finish_time->time > \OmegaUp\Time::get()
```

es bastante más común tener el operator en la línea anterior.

Reply:
Listo